### PR TITLE
Add S04E06

### PR DESCRIPTION
**Pull request by stephaniehobson**
#172

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Social sharing image is not showing up.

File: content/episodes/2019-02-18-S04E06-decentralize-it.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+    **Manoush Z.:** Eugen says he counts about 1.8 million registered users, although he concedes that only about 170,000 of those users seem to be active on the platform, but it's a start. The other advantage he points to is that Mastodon doesn't depend on user data and ad targeting to make money.
+
+    **Eugen Rochko:** Many of the reasons for Twitter's decision making going against user's interests is driven by its centralization. They have to perform financially. Now with a decentralized approach, we don't really need to monetize because the costs of operating everything are just so comparatively small for any given independent entity, right?
+
+    **:** For me, it's like, let's say about $500 per month to host my server of Mastodon, and that's housing 300,000 registered accounts. I have a Patreon that more than covers that. For many other administrators of Mastodon servers, it's the same situation.
```
Comment (craigcook) [inline content/episodes/2019-02-18-S04E06-decentralize-it.md]: Missing a name here.

Comment (craigcook) [reply content/episodes/2019-02-18-S04E06-decentralize-it.md]: Though on further thought and looking at the rest of the transcript, in some sections the speaker name is repeated at the start of each paragraph when it's still the same speaker, and in other places the same speaker goes for a few paragraphs with their name only appearing on the first one. I'm not sure which way is more standard but we should be consistent, within a single episode at least. From past episodes it looks like we've usually done the name once when the speaker changes so maybe we should stick with that format here?

File: content/episodes/2019-02-18-S04E06-decentralize-it.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+    Decentralization efforts are proof that the age of internet innovation is far from over. In fact, Mozilla staff work tirelessly on decentralized web standards, which have been — and continue to be — widely adopted.
+
+    Mozilla co-chaired the [W3C Social Web Working Group](https://www.w3.org/wiki/Socialwg#Social_Web_Working_Group) 2014 through 2018, which produced several key decentralized social web standards. Some have dozens of implementations like:
+
+    - [Webmention](https://indieweb.org/Webmention) (a standard for federating conversations across the
+    decentralized web); and
+    - [MicroPub](https://indieweb.org/Micropub) (a standard API for client applications to post to decentralized web services).
+
+    Check out [IndieWeb.org](https://spec.indieweb.org/) for more on key decentralized web
```
Comment (craigcook) [inline content/episodes/2019-02-18-S04E06-decentralize-it.md]: This links to the technical specs, should it go to the indieweb.org home page?

File: content/episodes/2019-02-18-S04E06-decentralize-it.md (hunk 1 above)
Comment (stephaniehobson) [reply content/episodes/2019-02-18-S04E06-decentralize-it.md]: I think it's a different speaker. I'll listen to the episode and see who is talking.

File: content/episodes/2019-02-18-S04E06-decentralize-it.md (hunk 2 above)
Comment (stephaniehobson) [reply content/episodes/2019-02-18-S04E06-decentralize-it.md]: The sentence is talking about the web standards specs but the wording is confusing. I've added a second link to try to make sure users end up where they expect to.

File: content/episodes/2019-02-18-S04E06-decentralize-it.md (hunk 1 above)
Comment (stephaniehobson) [reply content/episodes/2019-02-18-S04E06-decentralize-it.md]: Still Eugen.

**Comment by stephaniehobson** [comment]
Updated!

File: content/episodes/2019-02-18-S04E06-decentralize-it.md (hunk 1 above)
Comment (craigcook) [reply content/episodes/2019-02-18-S04E06-decentralize-it.md]: A few times in the first part of the transcript the speaker name is repeated for each paragraph when it's still the same speaker (I'm guessing this was transcribed in sections by different people). Nothing is broken and it's still understandable so not a real showstopper problem, just inconsistent. I'd be in favor of removing those repeated names in the first third of the transcript but it's up to you.

**Pull request merged**